addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2126,6 +2132,16 @@ body.editor_enable.editor_has_snippets {
                         &:hover {
                             border-color: mix($o-we-handles-accent-color, $o-we-sidebar-content-field-pressed-bg, .4);
                             background-color: $o-we-sidebar-content-field-pressed-bg;
+
+                            // TODO In master: restore `.oe_snippet_remove`.
+                            &.fa-trash {
+                                border-color: mix($o-we-color-danger, $o-we-sidebar-content-field-pressed-bg, .4);
+                                background-color: $o-we-color-danger;
```

Comment:
I wonder why this was removed in 15.0, it looks a lot better in red :shrug: :+1: